### PR TITLE
DevEx: Added user contact state validation

### PR DESCRIPTION
**Pull request by ericsorenson**

### CODE REVIEW

File: shared/src/business/entities/User.js [hunk 1]
```diff
@@ -79,7 +79,10 @@ const userValidation = {
       }),
       state: joi.when('countryType', {
         is: COUNTRY_TYPES.INTERNATIONAL,
-        otherwise: joi.string().max(100).required(),
+        otherwise: joi
+          .string()
+          .valid(...Object.keys(US_STATES))
```
Comment (rachaelparris) [inline shared/src/business/entities/User.js]: We have some other options in the state select in some places:
https://github.com/flexion/ef-cms/blob/ef04282d0ca25a2ee04ed64fc311d76eb920d256/web-client/src/views/Practitioners/PractitionerForm.jsx#L321
Should those be added?

Comment (ericsorenson) [reply shared/src/business/entities/User.js]: They've been added!

**Review by kkoskelin — APPROVED** [review]

**Pull request merged**